content/en/blog/_posts/2024-05-13-Gateway-API-v1.1/index.md

```diff
@@ -0,0 +1,175 @@
+title: "Gateway API v1.1: Feature Graduation and Other Updates"
+date: 2024-05-13T19:00:00-08:00
+slug: gateway-api-v1-1
+---
+
+![Gateway API Logo](gateway-api-logo.png)
+**Authors:** 
+GRPCRoute - Richard Belleville (Google), 
+ParentRef Port - Frank Budinsky (IBM),
+Gateway Client Certificate Validation - Arko Dasgupta (?),
+GAMMA - John Howard (Google),
+Conformance Profiles and Reports - Mattia Lavacca (Kong), Christine Kim (?),
+Session Persistence and BackendLBPolicy - Grant Spence (Red Hat)
+TLS Terminology Clarifications - Candace Holman (Red Hat),
+and other reviewer and release note contributors.
+
+In the wake of the GA release of Gateway API last October, the Kubernetes
+SIG Network buoyantly announces the v1.1 release
+of [Gateway API](https://gateway-api.sigs.k8s.io/). Gateway API features for
+**GRPCRoute**, **ParentRef Port**, and **Conformance Profiles and Reports** will
+graduate to
+Standard. In addition, **Gateway Client Certificate Verification** and **Session
+Persistence** are adopted as new, Experimental features. **Gateway API for
+Service Mesh (GAMMA)** support, and clarifications to TLS
+terminology and gateway merging are added to strengthen the release.
+
+## What's new
+
+### Graduation to Standard
+This release includes the graduation to Standard of four eagerly awaited features.
+This means they are no longer experimental concepts. **Standard** denotes a high
+level of confidence in the API surface and provides guarantees of backward
+compatibility.  Though these features are now considered stable, that does not
+mean that they are complete. These and other features will continue to evolve via
+the Experimental channel. For more information on how all of this works, refer
+to the [Gateway API Versioning Policy](https://gateway-api.sigs.k8s.io/concepts/versioning/).
+
+
+#### [GAMMA](https://gateway-api.sigs.k8s.io/mesh/)
+Service mesh support in Gateway API allows service mesh users to use the same
+API to manage ingress traffic and mesh traffic, reusing the same policy and
+routing interfaces. In Gateway API v1.1, routes (such as HTTPRoute) can now have
+a `Service` as a `parentRef`, to control how traffic to specific services behave.
+For more information, read the 
+[service mesh](https://gateway-api.sigs.k8s.io/mesh/) documentation or see the
+list of 
+[implementations](https://gateway-api.sigs.k8s.io/implementations/#service-mesh-implementation-status).
+
+Leading Contributors: Flynn, John Howard, Keith Mattix, and Mike Morris
+
+#### [GRPCRoute](https://gateway-api.sigs.k8s.io/guides/grpc-routing/)
+If you are already using the experimental version of GRPCRoute, we recommend holding
+off on upgrading to the standard channel version of GRPCRoute until the
+controllers you're using have been updated to support GRPCRoute v1. Until then,
+it is safe to upgrade to the experimental channel version of GRPCRoute in v1.1
+that includes both v1alpha2 and v1 API versions.
+
+Leading Contributor: Richard Belleville
```

Comment:
@gnossen please check and add/change anything you'd like via code suggestion.

Reply:
This looks great! Thanks @candita

Reply:
@gnossen completely optional, but if you want to add a simple example of GRPCRoute in action, that could be useful here for any readers that aren't already familiar with it.